### ISSUE TITLE
Given request parameters

### ISSUE THREAD

**Issue by CurroRodriguez** [behave-restful/behave-restful#25]
Implement support for query parameters in a request. The step implementation allows to define multiple parameters that will be passed:

```gherkin
Given request parameters
    | name | value |
    | a        | 1        |
    | b        | 20      |
```

**Comment by CurroRodriguez** [comment]
Once we support this, we should modify the invoker functions to pass those parameters to the session.

**Comment by CurroRodriguez** [comment]
Implemented in PR #32
